src/python/grpcio/grpc/_cython/_cygrpc/aio/common.pyx.pxi

```diff
@@ -165,3 +167,22 @@ async def generator_to_async_generator(object gen, object loop, object thread_po
 
     # Port the exception if there is any
     await future
+
+
+if PY_MAJOR_VERSION >=3 and PY_MINOR_VERSION >=7:
+    def get_working_loop():
+        """Returns a running event loop."""
+        return asyncio.get_running_loop()
+else:
+    def get_working_loop():
+        """Returns a running event loop.
+        
+        Due to a defect of asyncio.get_event_loop, its returned event loop might
+        not be set as the default event loop for the main thread. So, we will
+        raise RuntimeError if the returned event loop is not running.
+        """
+        loop = asyncio.get_event_loop()
+        if loop.is_running():
+            return loop
+        else:
+            raise RuntimeError('no running event loop')
```

Comment:
Suggestion: `"No running event loop detected. This function must be called from inside of a running event loop."`

Reply:
Updated to your version.

BTW, `RuntimeError('no running event loop')` is raised by `get_running_loop`. But your version is more useful for debugging.